paper-slider.html

```diff
@@ -543,6 +543,10 @@
         this._setRatio(this._calcRatio(this.immediateValue));
 
         this.$.sliderKnob.style.left = (this.ratio * 100) + '%';
+        if (this.dragging) {
+          this._knobstartx = this.ratio * this._w || 0;
```

Comment:
It was added [in the initial commit](https://github.com/PolymerElements/paper-slider/commit/c454112c6015dae4fad53da9405ee11ea1a186e9#diff-796ae68652c9d9635b1db7325e829a16R91).